app/helpers/versions_helper.rb

```diff
@@ -43,80 +43,65 @@ def show_past_versions(obj, args = {})
 
   def previous_version_link(previous_version, obj)
     str = :show_name_previous_version.t + " " + previous_version.version.to_i
-    initial_html(obj) +
+    initial_version_html(obj) +
       link_with_query(str,
                       controller: "#{obj.show_controller}/versions",
                       action: :show, id: obj.id,
                       version: previous_version.version) +
       safe_br
   end
 
-  def initial_html(obj)
+  def initial_version_html(obj)
     :VERSION.t + ": " + obj.version.to_s + safe_br
   end
 
   def build_version_table(obj, args)
     obj.versions.reverse.map do |ver|
-      [find_ver_date(ver), indent,
-       find_ver_user(ver), indent,
-       calc_link(obj, ver, args), indent]
+      [find_version_date(ver), indent,
+       find_version_user(ver), indent,
+       style_version_link(obj, ver, args), indent]
     end
   end
 
-  def calc_link(obj, ver, args)
-    link = query_link(obj, ver, initial_link(ver))
+  def style_version_link(obj, ver, args)
+    link = link_to_version(initial_version_link_text(ver), ver, obj)
     if args[:bold]&.call(ver)
       content_tag(:b, link)
     else
       link
     end
   end
 
-  def find_ver_date(ver)
+  def find_version_date(ver)
     ver.updated_at.web_date
   rescue StandardError
     :unknown.t
   end
 
-  def find_ver_user(ver)
+  def find_version_user(ver)
     user = User.safe_find(ver.user_id)
     return :unknown.t unless user
 
     user_link(user, user.login)
   end
 
-  def link_to_ver(link, ver, obj)
+  def link_to_version(text, ver, obj)
     if ver == obj.versions.last
-      link_with_query(link, controller: obj.show_controller,
+      link_with_query(text, controller: obj.show_controller,
                             action: obj.show_action,
                             id: obj.id)
     else
-      link_with_query(link,
+      link_with_query(text,
                       controller: "#{obj.show_controller}/versions",
                       action: :show, id: obj.id,
                       version: ver.version)
     end
   end
 
-  def initial_link(ver)
+  def initial_version_link_text(ver)
     link = "#{:VERSION.t} #{ver.version}"
     return link unless ver.respond_to?(:format_name)
```

Comment:
@nimmolo, @pellaea 
Under what circumstances could the conditional (`ver.respond_to?(:format_name)`) be true?
(I'm asking because I'd like to cover the next line, which is currently not covered.)

Reply:
@JoeCohen Under no circumstance I can figure. 

Name versions respond to 
 `text_name`: "Cabalodontia",
 `search_name`: "Cabalodontia M. Piatek",
 `display_name`: "**__Cabalodontia__** M. Piatek",
 `sort_name`: "Cabalodontia  M. Piatek",

I guess the question is, would any of these make this table more useful?

Reply:
Incidentally I wonder why Piątek lost the accent under the "a". That accent is more or less equivalent to a nasal "n" and is important in Polish...

Reply:
1. It would be slightly more useful -- provided you thing there's space -- to show for each version:
version #, date, user, Display Name
That would show the key changes in the table, without having to click through each of the versions.
BUT, I don't know if it's worth your time.
2. Good catch on Piątek. I fixed it. My guess is that the accent got dropped because of user error/carelessness.

Reply:
We can make space in the table. But i can't get the dang html tags in the display name to render.

Even after calling html_safe in about 5 different places. I despise dealing with that stuff! It takes me forever to figure out where to permit the tags.

<img width="751" alt="Screen Shot 2023-04-08 at 4 19 23 PM" src="https://user-images.githubusercontent.com/1948095/230746446-6076080e-6f5f-498a-b33e-d37200d7c307.png">

I can check it in and hand it over if you think you can figure it out.

Reply:
Oh hey, i got it. Checking it in now.